tests/xapi-plugins/test_zfs.py

```diff
@@ -0,0 +1,7 @@
+import json
+
+from tests.storage.zfs.conftest import zpool_vol0, host_with_zfs
+
+def test_list_zfs_pools(host, zpool_vol0):
+    res = host.call_plugin('zfs.py', 'list_zfs_pools')
+    assert json.loads(res).get("vol0") is not None
```

Comment:
I guess this will raise when there's no vol0, so the test won't work in that case?

Reply:
fixture `zpool_vol0` creates a `vol0`

Reply:
Ah, indeed